edx_notifications/server/web/static/edx_notifications/js/models/counter_icon_model.js

```diff
@@ -0,0 +1,11 @@
+var CounterIconModel = Backbone.Model.extend({
+    /* model for the Notification Icon which will display a counter of unread messages */
+
+    defaults: {
+        /* start with unknown number of unread notifications */
+        'count': null
+    },
+
+    /* REST API endpoint to use to get authenticated user's notification count */
+    url: "/api/edx_notifications/v1/consumer/notifications/count?unread=True&read=False"
```

Comment:
Our best practice is to pass the URL down from the server, rather than having it hard-coded on the client. Maybe that is less important going forward as we have very cleanly namespaced URLs like this one.